Entity/AudioSampleMetaData.py

```diff
@@ -26,6 +26,8 @@ class AudioSampleMetaData(Base):
     tone = Column(String(255))
     timestamp = Column(Integer)
     username = Column(String(255))
+    script = Column(String(1024))
```

Comment:
I am surprised that `1024` is allowable in MySQL

Reply:
The script is something I added with the latest version of the CSAI Recorder. Since we now know what people are saying given the prompt, we can store the script and use this training set in other NLP projects

Reply:
I'm not yet set up with the database. Lets get that working tomorrow